### ISSUE TITLE
[bug] DNS server query error

### ISSUE THREAD

**Issue by bobberb** [okTurtles/dnschain#54]
2014-10-08T16:31:25.170Z - error: [DNS] namecoin failed to resolve (dns.coffee:123)
{ err: '"401"<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"\r\n"http://www.w3.org/TR/1999/REC-html401-19991224/loose.dtd">\r\n<HTML>\r\n<HEAD>\r\n<TITLE>Error</TITLE>\r\n<META HTTP-EQUIV=\'Content-Type\' CONTENT=\'text/html; charset=ISO-8859-1\'>\r\n</HEAD>\r\n<BODY><H1>401 Unauthorized.</H1></BODY>\r\n</HTML>\r\n',
  result: undefined,
  q:
   { name: 'google.bit',
     type: 1,
     class: 1 } }

Arch x64, npm installed.  Namecoin running fine.

**Comment by cryptostorm** [comment]
I had the same problem, was getting that 401 Unauthorized error with any .bit I tried looking up via `dig`. Pretty sure the problem was that my namecoind and dnschain were running under separate accounts so dnschain couldn't read the namecoin.conf to get the rpc user/pass. To fix I ran dnschain with the command-line args --rpcuser=myuser and --rpcpassword=mypass (myuser/mypass as set in namecoin.conf) and now okturtles.bit seems to resolve as it should.
